src/coreclr/vm/bundle.cpp

```diff
@@ -47,6 +48,10 @@ Bundle::Bundle(LPCSTR bundlePath, BundleProbeFn *probe)
     size_t baseLen = pos - bundlePath + 1; // Include DIRECTORY_SEPARATOR_CHAR_A in m_basePath
     m_basePath.SetUTF8(bundlePath, (COUNT_T)baseLen);
     m_basePathLength = (COUNT_T)baseLen;
+
+    SString extractionPathMaybe;
+    if (HostInformation::GetProperty(HOST_PROPERTY_BUNDLE_EXTRACTION_PATH, extractionPathMaybe))
+        m_extractionPath.Set(extractionPathMaybe.GetUnicode());
```

Comment:
Do we ever do validation if this path is a directory or file? Is there some validation the path exists or is accessible?

Reply:
Not on the runtime side - the singlefilehost does. For me, this falls under the category of things where the runtime accepts that the host does the right thing (`APP_CONTEXT_BASE_DIRECTORY` is a similar example)